hooks/rcon_command.php

```diff
@@ -5,21 +5,21 @@
 session_name('CyberWorks');
 session_set_cookie_params(1209600);
 session_start();
-
 if (isset($_POST['sid']) && isset($_POST['command']) && isset($_SESSION['user_level'])) {
     if ($_SESSION['user_level'] > 3) {
         $db_connection = masterConnect();
-        $sid = clean($_POST['sid'], "int");
-        $rid = clean($_POST['id'], "int");
-        $cmd = clean($_POST['command'], "string");
+        $sid = clean((isset($_POST['sid'])) ? $_POST['sid'] : 0, "int");
```

Comment:
Oh yeah, it is, but there is no $_POST['id'] cover, (I did it because of an error about that)